sdk/identity/identity/src/index.ts

```diff
@@ -71,12 +71,12 @@ export {
   InteractiveBrowserCredentialInBrowserOptions,
   BrowserLoginStyle,
 } from "./credentials/interactiveBrowserCredentialOptions.js";
+export { ManagedIdentityCredential } from "./credentials/managedIdentityCredential/index.js";
 export {
-  ManagedIdentityCredential,
   ManagedIdentityCredentialClientIdOptions,
   ManagedIdentityCredentialResourceIdOptions,
   ManagedIdentityCredentialObjectIdOptions,
-} from "./credentials/managedIdentityCredential/index.js";
+} from "./credentials/managedIdentityCredential/options.js";
```

Comment:
But these are not used in browsers at all.  I wonder whether it is better to have a browser version of this file where NodeJS only types are not exported.

Reply:
This has been discussed offline.